aquacomputer_d5next.c

```diff
@@ -292,6 +292,24 @@ static int aqc_get_ctrl_val(struct aqc_data *priv, int offset)
 	return ret;
 }
 
+/* Refreshes the control buffer and returns u8 value at offset */
+static int aqc_get_u8_val(struct aqc_data *priv, int offset)
```

Comment:
Regarding these u8 functions (get and set), since they're pretty much the same as the original ones dealing with u16, please merge them into one, like [this](https://github.com/aleksamagicka/aquacomputer_d5next-hwmon/blob/656874a69ac4b35cd5d35ec091877c1d99e712fb/aquacomputer_d5next.c#L496).

Reply:
done in 66d24cb